fieldservice_contract/__manifest__.py

```diff
@@ -0,0 +1,30 @@
+# Copyright (C) 2019 - Akretion
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Field Service - Contracts',
+    'summary': 'Manage FSM Contracts',
+    'author': 'Akretion, '
+              'Odoo Community Association (OCA)',
+    'website': 'https://github.com/OCA/field-service',
+    'category': 'Field Service',
+    'license': 'AGPL-3',
+    'version': '12.0.1.0.0',
+    'depends': [
+        'contract',
```

Comment:
```suggestion
        'contract',
        'product_contract',
```

Need to add this dependency for sale order to create the contract